extmethods/blas/templates/methods.json

```diff
@@ -105,18 +107,20 @@
     "c": {
       "type":        "bh_complex64",
       "scalar_type": "bh_complex64",
+      "blas_type":   "bh_float32",
       "alpha":       "bh_complex64 alpha; alpha.real = 1.0; alpha.imag = 0.0;",
-      "alpha_arg":   "(void*) &alpha",
+      "alpha_arg":   "(bh_float32*) &alpha",
```

Comment:
Do we need this alpha_arg anymore? Or does it actually take a pointer to an float and then expects that the imaginary part is the next thing in memory? (2-array) Or could we just write 1.0?